### PR TITLE
Version tag showing

### PR DESCRIPTION
**Pull request by Johngtka**
Features List:
- [X] `added card in about component for displaying project version and another info`
- [x] `added displaying project (icon, title, version, authors and copyright info) in about card`
- [x] `added translations for info in card`

obraz

### CODE REVIEW

**Comment by macrog** [comment]
at this point we have that

| prod | dev | 
| --- | --- |
| image | image |

and what we want is something like this:
image

please add missing things, and style it so it looks more less like on above design

File: src/app/about/about.component.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { Component } from '@angular/core';
-import { environment } from '../../environments/environment.prod';
+import { environment } from '../../environments/environment';
```
Comment (macrog) [inline src/app/about/about.component.ts:2]: we have 3 files in environment folder:
image

always use `environment .ts` in any import!

File: src/app/services/patient-service.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { HttpClient } from '@angular/common/http';
 import { Observable } from 'rxjs';
 
 import { Patient } from '../models/patient';
-import { environment } from '../../environments/environment.dev';
+import { environment } from '../../environments/environment';
```
Comment (macrog) [inline src/app/services/patient-service.ts:7]: we have 3 files in environment folder:
image

always use `environment .ts` in any import!

File: src/app/services/store.service.ts [hunk 3]
```diff
@@ -4,7 +4,7 @@ import { HttpClient } from '@angular/common/http';
 import { Observable } from 'rxjs';
 
 import { Store } from '../models/store';
-import { environment } from '../../environments/environment.dev';
+import { environment } from '../../environments/environment';
```
Comment (macrog) [inline src/app/services/store.service.ts:7]: we have 3 files in environment folder:
image

always use `environment .ts` in any import!

File: src/app/services/visit.service.ts [hunk 4]
```diff
@@ -4,7 +4,7 @@ import { HttpClient } from '@angular/common/http';
 import { Observable } from 'rxjs';
 
 import { Visit } from '../models/visit';
-import { environment } from '../../environments/environment.dev';
+import { environment } from '../../environments/environment';
```
Comment (macrog) [inline src/app/services/visit.service.ts:7]: we have 3 files in environment folder:
image

always use `environment.ts` in any import!

**Comment by Johngtka** [comment]
S&M commit proposal:
`feat: project version improvement - displaying info (#)`

File: src/app/about/about.component.ts [hunk 5]
```diff
@@ -1,8 +1,13 @@
...
 
 @Component({
     selector: 'app-about',
     templateUrl: './about.component.html',
     styleUrls: ['./about.component.css'],
 })
-export class AboutComponent {}
+export class AboutComponent {
+    Title = environment.TITLE;
```
Comment (macrog) [inline src/app/about/about.component.ts:10]: why `Title` with uppercase? it should be `title`

File: src/app/about/about.component.ts [hunk 6]
```diff
@@ -1,8 +1,13 @@
 import { Component } from '@angular/core';
+import { environment } from '../../environments/environment';
```
Comment (macrog) [inline src/app/about/about.component.ts:2]: organize imports, add empty line in between different sections of imports

**Review by macrog — CHANGES_REQUESTED** [review]

**Comment by macrog** [comment]
Show the same icon that we use in the about page as a `favicon` ❗

**Comment by Johngtka** [comment]
> Show the same icon that we use in the about page as a `favicon` ❗

like this
obraz

File: src/app/about/about.component.html [hunk 7]
```diff
@@ -1,9 +1,14 @@
 <div class="appVersion">
     <mat-card>
         <mat-card-content>
-            <img class="icon" src="assets/img/magnet.svg" /><br />
+            <img
+                rel="icon"
+                class="icon"
+                type="image/x-icon"
```
Comment (macrog) [inline src/app/about/about.component.html:7]: what is this for?

**Review by macrog — APPROVED** [review]
👍

**Pull request merged**